cmd/ak/cmd/vars/set.go

```diff
@@ -11,10 +11,13 @@ import (
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
-var secret, optional bool
+var (
+	secret, optional bool
+	description      string
+)
 
 var setCmd = common.StandardCommand(&cobra.Command{
-	Use:     "set <key> [<value>] [--secret] [--optional] <--env=.. | --connection=....> [--project=...]",
+	Use:     "set <key> [<value>] [--secret] [--description <DESC>] [--optional] <--env=.. | --connection=....> [--project=...]",
```

Comment:
1. I honestly don't believe that anyone will ever use this flag in the CLI
2. This allows anyone to override an existing description of a var, which makes it even less useful as far as documentation goes